docs/pages/blog/lab-date-pickers-to-mui-x.md

```diff
@@ -8,12 +8,12 @@ card: true
 ---
 
 After more than 18 months in the lab, the date and time picker components have found a new home as part of MUI X.
-
```

Comment:
The next paragram starts with "This means". Shouldn't it be part of the same paragraphe?

Reply:
Ok, I see. I guess it was engineered this way then.